lib/Dialect/LWE/IR/LWEDialect.cpp

```diff
@@ -421,6 +473,19 @@ LogicalResult verifyEncodingAndTypeMatch(mlir::Type type,
     return success();
   }
 
```

Comment:
If the old encodings (`PolynomialEvaluationEncodingAttr` and `InverseCanonicalEmbeddingEncodingAttr`) have no more uses, can you just remove them?

Reply:
(can also wait until a follow-up PR that cleans up all the old stuff)